types/oauth/index.d.ts

```diff
@@ -5,14 +5,14 @@ import { Agent, ClientRequest, IncomingMessage, OutgoingHttpHeaders, RequestOpti
 import { UrlWithStringQuery } from "url";
 
 export type oauth1tokenCallback = (
-    err: { statusCode: number; data?: any },
+    err: Error | { statusCode: number; data?: any },
```

Comment:
reviewing purely from a backward-compatibility standpoint, the oauth2 change makes it easier to use; the oath1 change makes it harder to use because now you have to check whether you have an Error or the custom object type.